packages/react-table/scripts/createSubpaths.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Re-export subpath modules into the package root directory for ease of access.
```

Comment:
I don't think we can keep this description so simple anymore since there is a lot more going on than a single copySync.

Reply:
Do you have any recommendations?

It's doing a lot more operations, but it's still essentially completing the same goal of making the components available via importing from `@patternfly/react-core/deprecated` etc, it just does it via re-exporting rather than copying the actual component files.

Reply:
I don't, and maybe it's just that we need better comments around the operations that are taking place below.